File: Factorial.c
#include<stdio.h>
#include<stdlib.h>
int main()
{
    int i,n,f=1;
    printf("Enter number:");
    scanf("%d",&n);
    for(i=1;i<=n;i++)
    {
        f*=i;
    }
    printf("Factorial:%d",f);
    return(0);
}

File: factorial.java
import java.util.*;

public class prime
{
  public static void main(String args[])
  {

        Scanner scan = new Scanner(System.in);
        int n;
        int f=1;
        int i;


        System.out.println("Enter a Number : ");
        n = scan.nextInt();
        
        for(i=1; i<=n; i++)
        {
         f=f*i;
        }
        System.out.println("Factorial : " + f);
        
  }
 } 
